ArjunaJTS/interop/README.md

```diff
@@ -0,0 +1,430 @@
+OVERVIEW
+--------
+
+This example shows how to propagate a transaction context between the WildFly and GlassFish application
+servers. There is a single script to run the full example but you may find it more instructive to run
+the example in steps perhaps in separate command windows:
+
+USAGE
+-----
+
+./run.sh
+
+or to run each step at a time:
+
+step1.sh # build WildFly and GlassFish (including checking out the repo) and the demo EJBs
+step2.sh # configure WildFly to use JTS and start it
+step3.sh # start GlassFish
+step4.sh # deploy demo EJBs to WildFly and GlassFish
+step5.sh # perform a transactional EJB call from GlassFish to WildFly
+step6.sh # perform a transactional EJB call from WildFly to GlassFish 
+step7.sh # shutdown GlassFish and WildFly
+
+
+EXPECTED OUTPUT
+---------------
+
+This output is from running steps 2 to 7 (step 1 is is not show since it is quite verbose).
+
+The key lines to verify that the GlassFish to WildFly transactional EJB call worked is to look for the line:
+===== JTS interop quickstart: step 5 (EJB call gf -> wf):
+
+followed soon after by
+
+Next: 8000
+
+The key lines to verify that the WildFly to GlassFish transactional EJB call worked is to look for the line:
+===== JTS interop quickstart: step 6 (EJB call wf -> gf):
+
+followed soon after by
+
+Next: 7000
+
+An example of the full output for steps 2 to 7 is: 
```

Comment:
Agreed, I have removed the output. Now I just have:

EXPECTED OUTPUT
---------------

These steps generate too much output to show in full.

The key output lines to verify that the GlassFish to WildFly transactional EJB call worked is to look for the line:

&gt; ===== JTS interop quickstart: step 5 (EJB call gf -> wf):

followed soon after by a line that includes

&gt;  Next: 8000

The number printed will increment on each EJB call.

The key output lines to verify that the WildFly to GlassFish transactional EJB call worked is to look for the line:
&gt;  ===== JTS interop quickstart: step 6 (EJB call wf -> gf):

followed soon after by

&gt;  Next: 7000

The number printed will increment on each EJB call.